worlds/sc2wol/Client.py

```diff
@@ -323,10 +327,15 @@ def on_package(self, cmd: str, args: dict):
 
             self.mission_order = args["slot_data"].get("mission_order", 0)
             self.final_mission = args["slot_data"].get("final_mission", 29)
-            self.player_color = args["slot_data"].get("player_color", 2)
+            self.player_color = args["slot_data"].get("player_color_terran_raynor", 2)
```

Comment:
In the big refactor, I got Ziktofel to add a generation version number. Could update that and check it here for backwards compatibility on this option. Not critical here, as the worst that happens is some people transferring versions lose their colour, but good to get the pattern in here.

Reply:
I'm not really sure how compatible pre-HotS/cross-campaign worlds are with this rework, saving the color might be an exercise in futility that leaves a (permanent?) leftover.

Reply:
On second thought, this won't work at all because the game name will be changes from `Starcraft 2 Wings of Liberty` to just `Starcraft 2`, so old worlds won't be able to connect.